test/integration/aws/default/build/flow_log.tf

```diff
@@ -0,0 +1,68 @@
+resource "aws_flow_log" "fl-vpc" {
+  log_group_name = "${aws_cloudwatch_log_group.test_log_group.name}"
+  iam_role_arn   = "${aws_iam_role.test_role.arn}"
+  vpc_id         = "${data.aws_vpc.default.id}"
+  traffic_type   = "ALL"
+}
+
+resource "aws_flow_log" "fl-subnet" {
+  log_group_name = "${aws_cloudwatch_log_group.test_log_group.name}"
+  iam_role_arn   = "${aws_iam_role.test_role.arn}"
+  subnet_id      = "${aws_subnet.subnet_01.id}"
+  traffic_type   = "ALL"
+}
+
+output "fl-vpc" {
```

Comment:
Our fixture naming standard is <resource_type>_<fixture_name>_<property>.  So you might name this `flow_log_alpha_vpc_id` or `flow_log_vpc_vpc_id`.